builder/qemu/step_type_boot_command.go

```diff
@@ -146,6 +148,14 @@ func vncSendString(c *vnc.ClientConn, original string) {
 	shiftedChars := "~!@#$%^&*()_+{}|:\"<>?"
 	waitRe := regexp.MustCompile(`^<wait([0-9hms]+)>`)
 
+	// Send the key events. We add a 100ms sleep after each key event
```

Comment:
Yeah I could clarify this comment; it was copied from the earlier implementation.